@commitlint/cli/src/cli.test.ts

```diff
@@ -1,7 +1,7 @@
 import {fix, git} from '@commitlint/test';
 import execa from 'execa';
 import fs from 'fs-extra';
-import merge from 'lodash.merge';
+import merge from 'lodash/merge';
```

Comment:
I'd prefer the following personally.

```suggestion
import { merge } from 'lodash';
```